.github/workflows/build_dependency_ubuntu-22.04.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# The package list is designed for Ubuntu 20.04 LTS
```

Comment:
As discussed, I believe the script should be merged to the [install_dependencies.sh](https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/7a440a99b2efa59a9832540920156ca792093bab/.github/workflows/test.yml#L121)

Also the dependencies in https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/7a440a99b2efa59a9832540920156ca792093bab/.github/scripts/hostsetup.sh#L23-L72

should be updated as well.